### PR TITLE
Pre train results

### PR DESCRIPTION
**Pull request by pshivraj**
This PR adds Inference notebook for preliminary pre-trained models result on validation data.

### CODE REVIEW

**Comment by lmtoan** [comment]
Thanks Puru.

For the image with falling bottles, I think we achieved the objective of fixing the alignment recognition. But for other images, there seems to be a lot of false positives in the background or on container labels. How does mAP metric penalize false positives?

Since this method is working (overfitting) for bottles, I have a lot of hopes that we can improve performance on cereal box and chip bags dramatically. Right now, COCO pre-trained does not do so well.

**Review by lmtoan — COMMENTED** [review]
I can work on arranging the before (COCO weights) and after (CloMask weights) image arrangement for better visual
